### PR TITLE
ycsb: Fix rate limiting and resetting of stats after loading phase

### PR DESCRIPTION
**Pull request by a-robinson**

### CODE REVIEW

**Comment by a-robinson** [comment]
Combined, these two changes take us from post-data-loading output like this:

```
Loading complete: 7.0s
     0s       1305.9           6 / 0 / 0        1301 / 0           0 / 0
     1s        128.7         124 / 0 / 0           5 / 0           0 / 0
     2s        128.8         125 / 0 / 0           4 / 0           0 / 0
     3s        131.9         124 / 0 / 0           8 / 0           0 / 0
     4s        132.3         124 / 0 / 0           8 / 0           0 / 0
     5s        132.4         126 / 0 / 0           6 / 0           0 / 0
```

To output like this:

```
Loading complete: 6.8s
     0s        297.6          48 / 0 / 0           6 / 0           0 / 0
     1s        300.0         282 / 0 / 0          18 / 0           0 / 0
     2s        300.0         283 / 0 / 0          17 / 0           0 / 0
     3s        299.9         281 / 0 / 0          19 / 0           0 / 0
     4s        300.1         285 / 0 / 0          15 / 0           0 / 0
     5s        299.5         283 / 0 / 0          17 / 0           0 / 0
```

When running `./ycsb --concurrency=3 --rate-limit=100`

File: ycsb/main.go [hunk 1]
```diff
@@ -74,7 +76,7 @@ var writeDuration = flag.Duration("write-duration", 0,
 var verbose = flag.Bool("v", false, "Print *verbose debug output")
 var drop = flag.Bool("drop", true,
 	"Drop the existing table and recreate it to start from scratch")
-var rateLimit = flag.Uint64("rate-limit", 0,
+var maxRate = flag.Uint64("rate-limit", 0,
 	"Maximum number of operations per second per worker. Set to zero for no rate limit")
```
Comment (petermattis) [inline ycsb/main.go]: In `kv`, the rate limit is global, not per worker. We should probably do the same here for consistency. I think you just have to pass in a shared `*rate.Limiter` to `newYcsbWorker`.

**Review by petermattis — APPROVED** [review]
LGTM

File: ycsb/main.go (hunk 1 above)
Comment (rjnn) [reply ycsb/main.go]: +1

**Review by rjnn — APPROVED** [review]
LGTM

File: ycsb/main.go (hunk 1 above)
Comment (a-robinson) [reply ycsb/main.go]: Done.

**Comment by a-robinson** [comment]
Comment addressed. I also changed the name of the flag to match the identical flag in kv. Normally I wouldn't change a flag name, but given that this is also a big change in behavior it seems like a good time to move it.

**Review by petermattis — APPROVED** [review]

**Pull request merged**